pytential/symbolic/matrix.py

```diff
@@ -26,14 +26,14 @@
 """
```

Comment:
@alexfikl Could you review this file (if you haven't already)?

Reply:
I was the one who ported this (and the `proxy` stuff and associated tests), so they should be fine by me, but I'll give it another stare.

Let me know if you see anything fishy in there!

Reply:
Thanks! I've looked both files over, too, just now, and I didn't see anything anything alarming.